ts/routes/deck-options/FsrsOptions.svelte

```diff
@@ -148,7 +148,13 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
                         ignoreRevlogsBeforeMs: getIgnoreRevlogsBeforeMs(),
                         currentParams: params,
                     });
-                    if (
+
+                    if (!resp.fsrsItems) {
```

Comment:
Is checking the number of fsrsItems in Svelte reliable (vs checking them in Rust)? Asking just because there was an issue in the past where Anki would report 0 reviews even where the preset had some.

See
- https://github.com/ankitects/anki/issues/3242
- https://github.com/ankitects/anki/pull/3591

Reply:
I've made it so that it reporting no reviews wont prevent it from setting new parameters.